GitHub-app/src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from "redux";
+import infoReducer from "./infoReducer";
+
+export default combineReducers({
+  info: infoReducer,
+  repos: infoReducer,
+  isFetching: infoReducer
+});
```

Comment:
Trzykrotnie powielasz swoje reducery i w efekcie to co trzymasz w store. Mam wrażenie, że nie rozumiesz do końca jak działają reducery.